### PR TITLE
sb_categoryitem/item_sorting_widget_new

### PR DESCRIPTION
**Pull request by Tim-M-S**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested by the author
- [x] Changes have been tested by the reviewer
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: src/Widgets/Category/ItemSortingWidget.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+use Ceres\Widgets\Helper\BaseWidget;
+use Ceres\Helper\SearchOptions;
+
+class ItemSortingWidget extends BaseWidget
+{
+    protected $template = "Ceres::Widgets.Category.ItemSortingWidget";
+
+    protected function getTemplateData($widgetSettings, $isPreview)
+    {
+        $temp = [];
```
Comment (stentrop) [inline src/Widgets/Category/ItemSortingWidget.php]:
```suggestion
```

Comment (stentrop) [reply src/Widgets/Category/ItemSortingWidget.php]: Wird nur im if gebraucht, dann müssen wir nicht vorher den Platz reservieren

**Review by stentrop — CHANGES_REQUESTED** [review]
Die Fehler beheben wir besprochen

File: src/Widgets/Category/ItemSortingWidget.php (hunk 1 above)
Comment (Tim-M-S) [reply src/Widgets/Category/ItemSortingWidget.php]: done

**Review by stentrop — APPROVED** [review]

**Comment by maxhelligeplenty** [comment]
Hallo  ,
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested by the author
- [ ] Changes have been tested by the reviewer
- [x] Plugin can be built

@plentymarkets/ceres-io 
You can view, comment on, or merge this pull request online at:

  https://github.com/plentymarkets/plugin-ceres/pull/1606

-- Commit Summary --

  * cleaned pull request

-- File Changes --

    M contentWidgets.json (149)
    M resources/lang/de/Widget.properties (4)
    M resources/lang/en/Widget.properties (4)
    M resources/scss/ceres/widgets/Category/_toolbar-widget.scss (4)
    A resources/views/Widgets/Category/ItemSortingWidget.json (113)
    A resources/views/Widgets/Category/ItemSortingWidget.twig (31)
    A src/Widgets/Category/ItemSortingWidget.php (45)

-- Patch Links --

https://github.com/plentymarkets/plugin-ceres/pull/1606.patch
https://github.com/plentymarkets/plugin-ceres/pull/1606.diff

-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/plugin-ceres/pull/1606


Test.test.test

**Comment by maxhelligeplenty** [comment]
Hallo  ,
stentrop commented on this pull request.



> @@ -0,0 +1,45 @@
+
+
+namespace Ceres\Widgets\Category;
+
+use Ceres\Config\CeresConfig;
+use Ceres\Widgets\Helper\BaseWidget;
+use Ceres\Helper\SearchOptions;
+
+class ItemSortingWidget extends BaseWidget
+{
+    protected $template = "Ceres::Widgets.Category.ItemSortingWidget";
+
+    protected function getTemplateData($widgetSettings, $isPreview)
+    {
+        $temp = [];

```suggestion
```

-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/plugin-ceres/pull/1606#pullrequestreview-311065995

Test.test.test

**Comment by maxhelligeplenty** [comment]
Hallo  ,
stentrop commented on this pull request.



> @@ -0,0 +1,45 @@
+
+
+namespace Ceres\Widgets\Category;
+
+use Ceres\Config\CeresConfig;
+use Ceres\Widgets\Helper\BaseWidget;
+use Ceres\Helper\SearchOptions;
+
+class ItemSortingWidget extends BaseWidget
+{
+    protected $template = "Ceres::Widgets.Category.ItemSortingWidget";
+
+    protected function getTemplateData($widgetSettings, $isPreview)
+    {
+        $temp = [];

Wird nur im if gebraucht, dann müssen wir nicht vorher den Platz reservieren

-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/plugin-ceres/pull/1606#discussion_r342056136

Test.test.test

**Comment by maxhelligeplenty** [comment]
Hallo  ,
stentrop requested changes on this pull request.

Die Fehler beheben wir besprochen



-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/plugin-ceres/pull/1606#pullrequestreview-311078236

Test.test.test

**Comment by maxhelligeplenty** [comment]
Hallo  ,
Tim-M-S commented on this pull request.



> @@ -0,0 +1,45 @@
+
+
+namespace Ceres\Widgets\Category;
+
+use Ceres\Config\CeresConfig;
+use Ceres\Widgets\Helper\BaseWidget;
+use Ceres\Helper\SearchOptions;
+
+class ItemSortingWidget extends BaseWidget
+{
+    protected $template = "Ceres::Widgets.Category.ItemSortingWidget";
+
+    protected function getTemplateData($widgetSettings, $isPreview)
+    {
+        $temp = [];

done

-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/plugin-ceres/pull/1606#discussion_r342059701

Test.test.test

**Pull request merged**